site/docs/pt/guide/README.md

```diff
@@ -0,0 +1,41 @@
+![grammY](/images/grammY.webp)
+
+# Visão Geral
+
+Bem-vindo ao grammY!
+Você veio para o lugar certo.
+
+## O Que É o grammY?
+
+O grammY é um framework para criar bots do telegram.
+Ele pode ser usado com TypeScript e JavaScript e roda em Node.js, Deno, e no navegador.
+
+Dê uma olhada na [Introdução](./introduction) se você é novo no mundo dos bots do Telegram, especialmente se você tem pouca experiência com programação.
+
+Se você sabe como escrever uma aplicação em Node.js ou Deno ou já criou algum bot do Telegram com outro framework para bots, comece em poucos minutos com o nosso guia de [Primeiros Passos](./getting-started).
+
+## Sobre a documentação
+
+A documentação do grammY é dividida em três camadas.
+
+1. Documentação do grammY de alto nível _(this website)_
+2. [Referência da API de baixo nível do grammY](https://deno.land/x/grammy/mod.ts)
+3. [Referência da API HTTP bruta do Telegram](https://core.telegram.org/bots/api)
+
+**A primeira parte** (você está olhando pra ela!) explica como os bots funcionam e como usar o grammY.
+É o que você vai ver com mais frequência.
+A seção _Aprenda_ é sempre um bom start.
+Mas também passe pela nossa grande coleção de _Plugins_, e dê uma olhada nos _Exemplos_.
+
+**A segunda parte** é a [referência da API do grammY](https://deno.land/x/grammy/mod.ts), linkado no topo da página.
+É uma visão geral bem detalhada de cada pedacinho de código que o grammY fornece.
+É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis das dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

Comment:
```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis de dicas de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

Idk, this still feels weird. Tbf I've never seen tooltip translated like this 🤔

Reply:
yeah, I've spent a few minutes trying to find the best fit for it. Still weird. We could flag that to think about later.

Reply:
```suggestion
É gerada automaticamente a partir do código do grammY e contém todas as explicações úteis de descrições de ferramenta, que são geralmente encontradas quando você passa seu cursor sobre um elemento do grammY em um editor de código.
```

maybe like that? Or rephrase the entire sentence to find a better arrangement of words with the same meaning?